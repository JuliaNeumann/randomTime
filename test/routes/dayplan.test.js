import { shallow } from 'vue-test-utils';
import Dayplan from '../../app/routes/dayplan/dayplan.vue';

describe('Dayplan', () => {
  const wrapper = shallow(Dayplan);

  it('renders the correct markup', () => {
    expect(wrapper.element).toMatchSnapshot();
  })

});