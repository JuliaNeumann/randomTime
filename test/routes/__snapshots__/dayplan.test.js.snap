// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dayplan renders the correct markup 1`] = `
<div
  class="uk-container"
  id="app"
>
  <!---->
   
  <div>
    <h1
      class="uk-heading-divider"
    >
      Get Day Plan
    </h1>
     
    <form
      class="uk-form-stacked"
    >
      <fieldset
        class="uk-fieldset"
      >
        <label
          class="uk-form-label"
          for="number-of-hours"
        >
          Number of hours to plan:
        </label>
         
        <div
          class="uk-form-controls"
        >
          <input
            class="uk-input uk-form-width-medium"
            id="number-of-hours"
            type="number"
          />
        </div>
         
        <input
          class="uk-button uk-button-primary uk-margin"
          type="submit"
          value="Get"
        />
      </fieldset>
    </form>
     
    <!---->
     
    <!---->
     
    <!---->
     
    <!---->
  </div>
</div>
`;
